src/string-logger.js

```diff
@@ -4,14 +4,16 @@ import escapeRegExp from "lodash/escapeRegExp";
 import Logger from "./logger.js";
 import {getPathSeparator} from "./get-path-separator.js";
 
+import type {ILog} from "./types.js";
+
 /**
  * Rather than directly use this implementation, we wrap it with the main
  * Logger class so that we also capture the format calls as part of the
  * captured output. This allows us to have standardized format whether it is
  * invoked by the Logger implementation or someone calling Format/chalk before
  * passing it to a log call.
  */
-class StringLoggerInternal {
+class StringLoggerInternal implements ILog {
```

Comment:
Actually could just implement IStandardLog and drop the two additional methods I added.